js/repl/src/PresetLoadingAnimation.js

```diff
@@ -0,0 +1,56 @@
+import { css } from 'glamor';
+import React from 'react';
+
+const PresetLoadingAnimation = () =>
+  <div className={styles.loadingAnimation}>
+    <div className={`${styles.loadingTick} ${styles.loadingTick1}`} />
+    <div className={`${styles.loadingTick} ${styles.loadingTick2}`} />
+    <div className={`${styles.loadingTick} ${styles.loadingTick3}`} />
+    <div className={`${styles.loadingTick} ${styles.loadingTick4}`} />
+    <div className={`${styles.loadingTick} ${styles.loadingTick5}`} />
+  </div>;
+
+export default PresetLoadingAnimation;
+
+const bounce = css.keyframes({
+  '0%': { transform: 'scaleY(0.25)' },
+  '40%': { transform: 'scaleY(0.75)' },
+  '80%': { transform: 'scaleY(0.25)' },
+  '100%': { transform: 'scaleY(0.25)' }
+});
+
+const styles = {
+  loadingAnimation: css({
```

Comment:
The reason that I was asking was that some other solutions need you to extract the CSS into a regular CSS file for best performance. See this benchmark from Emotion, for example: https://github.com/tkh44/emotion/blob/master/docs/benchmarks.md. I wasn't sure if Glamor has the same perf characteristics.

>  I'm using it for the new React website too.
Is the new React website using server rendering with CSS-in-JS? That's an interesting approach, I haven't seen anyone do that before 😛  Or does the site require JS just to render static pages?